src/screens/SpeedUpAndCancelSheet.js

```diff
@@ -25,8 +25,15 @@ import {
   SlackSheet,
 } from '../components/sheet';
 import { Emoji, Text } from '../components/text';
-import { TransactionStatusTypes } from '@rainbow-me/entities';
-import { getProviderForNetwork, toHex } from '@rainbow-me/handlers/web3';
+import {
+  EIP1559TransactionType,
```

Comment:
this file should be using `EIP1559TransactionTypes` not `EIP1559TransactionType`